src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -221,8 +221,7 @@ public virtual ObjectResult StatusCode([ActionResultStatusCode] int statusCode,
         }
 
         /// <summary>
-        /// Creates a <see cref="ContentResult"/> object with <see cref="StatusCodes.Status200OK"/> by specifying a
-        /// <paramref name="content"/> string.
+        /// Creates a <see cref="ContentResult"/> object by specifying a <paramref name="content"/> string.
```

Comment:
I'm really not sure that this is more clear? Should say that unless you customize it the result is a 200?

Reply:
> Should say that unless you customize it the result is a 200?

More like if not customized it's whatever the server \ middleware pipeline initialized it to? Seems like more details than necessary.